### DISCUSSION TITLE
How many parameters can I use in dispatchReactUnityEvent()?

### DISCUSSION THREAD

**Discussion by hedgehoghash** [jeffreylanters/react-unity-webgl#477] [Question and Answer] [answered]
I'm curious if there's a limit to how many parameters can be used in this dispatch function when sending an event from Unity to React?

**Comment by jeffreylanters** [comment, marked as answer]
Not that I've added, but I'm sure there is a hardware limit haha.

**Reply by hedgehoghash** [reply]
Perfect, I don't think we'll get anywhere near the hardware limit. Just wanted to make sure there wasn't another limit to be aware of. Thanks!
